.travis.yml

```diff
@@ -1,41 +1,43 @@
 language: php
 
-php:
-  - 7.1
-  - 7.2
-
 env:
   global:
-    - setup=basic
+    - SETUP=stable
 
 matrix:
+  fast_finish: true
   include:
-    - php: 7.1
-      env: setup=lowest
-    - php: 7.1
-      env: setup=stable
     - php: 7.2
-      env: setup=lowest
     - php: 7.2
-      env: setup=stable
+      env: SETUP=lowest
+    - php: 7.3
+    - php: 7.3
+      env: SETUP=lowest
```

Comment:
Wait, this seems odd. Why are PHP 7.2 and 7.3 both marked as "SETUP=lowest"? Seems like one should be "stable" and one is "lowest".

Also, I don't see why there should be more than one version of 7.2 or 7.3.

Reply:
See other comment.